mks/model/stuf_02_04.py

```diff
@@ -314,7 +320,38 @@ def extract_verbintenis_data(persoon_tree: Tag):
 
 
 def extract_address(persoon_tree: Tag, is_amsterdammer):
-    result = []
+    addresses_vbl_source = persoon_tree.find_all("PRSADRVBL")
+    addresses_cor_source = persoon_tree.find_all("PRSADRCOR")
+    addresses_vbl = format_addresses(
+        addresses_vbl_source, ADDR_VBL, is_amsterdammer)
+    addresses_cor = format_addresses(
+        addresses_cor_source, ADDR_COR, is_amsterdammer)
+    addresses = merge_sort_addresses(addresses_vbl, addresses_cor)
+
+    return addresses
+
+
+def merge_sort_addresses(*address_lists):
+    all_addresses = list(chain(*address_lists))
+    if is_nil(all_addresses):
+        return {}, []
+    all_addresses.sort(
```

Comment:
Volgens mij sorteerd deze functie DESC toch? Dus het adres dat je als laatste ging bewonen kom je als eerste tegen in de lijst. Als er geen beginDatumVerblijf is wordt deze achteraan gezet. Daar kunnen we ook niks mee.